README.md

```diff
@@ -24,6 +30,9 @@ Use the Go CLI to run the application.
 Use the Go CLI to execute unit tests.
 `go test`
 
+## Adding new APIs
+MatterhornApiService follows a spec-first approach to defining API endpoints. To add, remove, or modify an API, first update the OpenAPI spec in `api/openapi.yaml`. Next, execute the VSCode task named "Generate Server Stubs". This will regenerate server stubs based on the updated OpenAPI spec without overwriting any existing implementation (as configured in `.openapi-generator-ignore`). You will need to modify `main.go` to create your *Service and *Controller structs and initialize them with any injected dependencies.
```

Comment:
is this api actually living at that path? We have a separate repo for it now, right?

Reply:
Right, I'll amend this.